srcpkgs/mutter/template

```diff
@@ -18,12 +18,11 @@ short_desc="Wayland display server, X11 window manager and compositor library"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Projects/Mutter/"
-#changelog="https://gitlab.gnome.org/GNOME/mutter/-/raw/gnome-45/NEWS"
```

Comment:
Add this back, uncomment it and make it `https://gitlab.gnome.org/GNOME/mutter/-/raw/gnome-46/NEWS`